src/guide/conditional.md

```diff
@@ -57,38 +57,38 @@ The `v-else-if`, as the name suggests, serves as an "else if block" for `v-if`.
   C
 </div>
 <div v-else>
-  Not A/B/C
+  Si no es A/B/C
 </div>
 ```
 
-Similar to `v-else`, a `v-else-if` element must immediately follow a `v-if` or a `v-else-if` element.
+Similar a `v-else`, un elemento `v-else-if` debe seguir inmediatamente a un elemento `v-if` o `v-else-if`.
 
 ## `v-show`
 
-Another option for conditionally displaying an element is the `v-show` directive. The usage is largely the same:
+Otra opción para mostrar un elemento condicionalmente es la directiva `v-show`. La utilización es básicamente la misma:
 
 ```html
-<h1 v-show="ok">Hello!</h1>
+<h1 v-show="ok">¡Hola!</h1>
 ```
 
-The difference is that an element with `v-show` will always be rendered and remain in the DOM; `v-show` only toggles the `display` CSS property of the element.
+La diferencia es que un elemento con `v-show` siempre se renderizará y permanecerá en el DOM; `v-show` simplemente alterna la propiedad CSS `display` del elemento.
 
-`v-show` doesn't support the `<template>` element, nor does it work with `v-else`.
+`v-show` no es compatible con elementos `<template>`, ni tampoco funciona con `v-else`.
 
 ## `v-if` vs `v-show`
 
-`v-if` is "real" conditional rendering because it ensures that event listeners and child components inside the conditional block are properly destroyed and re-created during toggles.
+`v-if` es una renderización condicional “real” ya que garantiza que los eventos y componentes secundarios dentro del bloque condicional sean debidamente destruidos y recreados durante la alternancia.
```

Comment:
I think `recreados` is not a good option here. What do you think about this?
```suggestion
`v-if` es una renderización condicional “real” ya que garantiza que los eventos y componentes secundarios dentro del bloque condicional sean debidamente destruidos y creados nuevamente durante la alternancia.
```

Reply:
Yes, `creados nuevamente` it's a better choice for this. :smile: